Formula/speex.rb

```diff
@@ -13,11 +13,19 @@ class Speex < Formula
 
   depends_on "pkg-config" => :build
   depends_on "libogg" => :recommended
+  depends_on "speexdsp" => :optional
 
   def install
     ENV.deparallelize
-    system "./configure", "--disable-debug", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
+    ENV["SPEEXDSP_CFLAGS"] = "-I#{HOMEBREW_PREFIX}/include/speex" if build.with? "speexdsp"
+    ENV["SPEEXDSP_LIBS"] = "-L#{HOMEBREW_PREFIX}/lib -lspeexdsp" if build.with? "speexdsp"
+    args = %W[
+      --prefix=#{prefix}
+      --disable-debug
+      --disable-dependency-tracking
+      --enable-sse
```

Comment:
Your description of why SSE should be enabled, if it's not the default, is not clear.